### ISSUE TITLE
Full screen limitations

### ISSUE THREAD

**Issue by andyneff** [DavidKMagnus/projector-hotkeys#3]
Your second limitation says it only support fullscreen. This can be easily fixed by allowing the form data to go down to -1, and never subtracting 1 from the monitor output if it's < 0.

(The first limitation really annoyed me too, so I got a little creative. I work in python (just for me), so I don't know what all is possible in lua, but you _can_ make it work if you use the window32 API a little, but of course this won't be portable to linux or mac, so you may be less interested in that... but for me, it feel really cool to be able to "peak" at a camera, and never leave the game.)

**Comment by Avasam** [comment]
@andyneff Could you make a PR of your fork please? I don't think I need this myself, but others might.

**Comment by DavidKMagnus** [comment]
Yes, definitely true that this is currently limited to full screen.
That was done mostly because I didn't need windowed projectors for myself and didn't want to create the UI to put in the positions. When opening a projector, the OBS API does allow you to provide coordinates, I just haven't done it yet.

**Comment by andyneff** [comment]
@Avasam Unfortunately, in my fork I rewrote it _all_ in python (as I do not know LUA), and it is not something you can PR into the LUA script. But thank you for the encouragement ^_^

Additionally, because the OBS API does not provide a way to close a windowed projector, I had to use Windows specific API calls to close the windowed projector, and without an X/Wayland and macOS equivalent, my work just isn't universal enough to be PRed. My fork can provide a proof of concept for anyone who knows these APIs well enough to come up with a similar close workaround.

@DavidKMagnus The Coordinate stuff was super obfuscated. IIRC there's a QT "Geometry" type that you have to manually make a base64 packed string [here](https://github.com/andyneff/projector-hotkeys/blob/main/projector.py#L130-L188) to set the coordinates with.
